webpack.config.js

```diff
@@ -13,6 +13,10 @@ module.exports = {
       './inventory/src/scss/itemIndex.scss',
     ],
     cartIndex: ['./inventory/src/js/cartIndex.js'],
+    addItem: [
+      './inventory/src/js/addItem.js',
+      './inventory/src/scss/addItem.scss',
```

Comment:
actually no need to include the scss file if it's not being used or needed, can delete the `addItem.scss` file if you feel its not going to be used in the future too